src/components/marker-table/index.js

```diff
@@ -35,22 +35,24 @@ type MarkerDisplayData = {|
 |};
 
 class MarkerTree {
-  _markers: Marker[];
+  _filteredMarkers: Marker[];
+  _allMarkers: Marker[];
```

Comment:
Perhaps this could be called `unfilteredMarkers` here so that there is a clear way to call this marker list.